classes/MedraWebservice.php

```diff
@@ -150,8 +142,8 @@ function _doRequest($action, $arg, $attachment = null) {
 			return $result;
 		}
 
-		if ($result === true && ($status = $response->getStatusCode()) != MEDRA_WS_RESPONSE_OK) {
-			$result = 'OJS-mEDRA: Expected ' . MEDRA_WS_RESPONSE_OK . ' response code, got ' . $status . ' instead.';
+		if ($result === true && ($status = $response->getStatusCode()) != self::MEDRA_WS_RESPONSE_OK) {
```

Comment:
I think `$result` here will always be true because it's only set to something other than `true` inside the catch block.